package.json

```diff
@@ -4,7 +4,8 @@
     "gulp": "^3.8.8"
   },
   "dependencies": {
-    "laravel-elixir": "^4.0.0",
-    "bootstrap-sass": "^3.0.0"
+    "@dosomething/forge": "^6.6.2",
+    "dosomething-modal": "^0.3.0",
```

Comment:
Wasn't entirely sure, but figured it'd be useful to add it. We can always remove later if not needed, but I have a feeling it might come in handy :)